Gems/Terrain/Assets/Shaders/Terrain/ClipmapComputeHelpers.azsli

```diff
@@ -18,23 +20,23 @@
 
 float GetClipmapScale(uint clipmapLevel)
 {
-    return PassSrg::m_clipmapData.m_clipmapScaleInv[clipmapLevel].x;
+    return TerrainSrg::m_clipmapData.m_clipmapScaleInv[clipmapLevel].x;
 }
 
 float2 GetPreviousClipmapCenter(uint clipmapLevel)
 {
-    return PassSrg::m_clipmapData.m_clipmapCenters[clipmapLevel].xy;
+    return TerrainSrg::m_clipmapData.m_clipmapCenters[clipmapLevel].xy;
 }
 
 float2 GetCurrentClipmapCenter(uint clipmapLevel)
 {
-    return PassSrg::m_clipmapData.m_clipmapCenters[clipmapLevel].zw;
+    return TerrainSrg::m_clipmapData.m_clipmapCenters[clipmapLevel].zw;
 }
 
 float2 GetWorldPosition(float2 clipmapCenter, uint2 pixelPosition, uint clipmapLevel)
```

Comment:
Comment added. It is to get the world position from a clipmap pixel.
Because clipmap is using toroidal addressing, the center of the clipmap is not always at middle. We need the center to do addressing. It is passed as a parameter because it can also use the previous center.